content/en/docs/Reference/declarative-config.md

```diff
@@ -0,0 +1,341 @@
+---
+title: "Declarative Config"
+linkTitle: "Declarative Config"
+weight: 4
+date: 2021-07-29
+---
+
+Declarative Config (DC) is the latest iteration of OLM's index format. It is a fully plaintext-based (JSON or YAML) evolution of the previous sqlite database format that is fully backwards compatible.
+
+## Motivation
+The primary motivations for this new format are to enable index editting, composability, and extensibility. 
+
+### Editting
+
+With DC, users interacting with the contents of an index are able to make direct changes to the index format and verify that their changes are valid. 
+
+Because the index is now stored in plaintext rather than an sqlite database, index maintainers can update an index without the use of a custom index-aware tool, like `opm`.
```

Comment:
```suggestion
Because the index is now stored in plaintext rather than a sqlite database, index maintainers can update an index without the use of a custom index-aware tool, like `opm`.
```